Explorer/Assets/DCL/Settings/ModuleControllers/SceneDistanceSettingsController.cs

```diff
@@ -20,17 +20,23 @@ public SceneDistanceSettingsController(SettingsSliderModuleView view, RealmParti
 
             view.SliderView.Slider.onValueChanged.AddListener(SetSceneDistanceSettings);
             SetSceneDistanceSettings(view.SliderView.Slider.value);
+
+            realmPartitionSettingsAsset.OnMaxLoadingDistanceInParcelsChanged += OnMaxLoadingDistanceInParcelsChangedChanged;
         }
 
-        private void SetSceneDistanceSettings(float distance)
-        {
+        private void SetSceneDistanceSettings(float distance) =>
             realmPartitionSettingsAsset.MaxLoadingDistanceInParcels = (int)distance;
-            settingsDataStore.SetSliderValue(SCENE_DISTANCE_DATA_STORE_KEY, distance, save: true);
+
+        private void OnMaxLoadingDistanceInParcelsChangedChanged(int newDistance)
+        {
+            view.SliderView.Slider.value = newDistance;
+            settingsDataStore.SetSliderValue(SCENE_DISTANCE_DATA_STORE_KEY, newDistance, save: true);
         }
 
         public override void Dispose()
         {
             view.SliderView.Slider.onValueChanged.RemoveListener(SetSceneDistanceSettings);
+            realmPartitionSettingsAsset.OnMaxLoadingDistanceInParcelsChanged -= OnMaxLoadingDistanceInParcelsChangedChanged;
```

Comment:
Similar to the landscape, can the scope of this change if we change realm?

Reply:
Same explanation than [here](https://github.com/decentraland/unity-explorer/pull/637#discussion_r1574750569), but in this case with the `realmPartitionSettingsAsset` scriptable object.